lib/utils/ConfigParser.js

```diff
@@ -290,12 +290,22 @@ class ConfigParser {
 
         for (let pattern of patterns) {
             let filenames = glob.sync(pattern)
+            
+            let patternExtension = ''
+            if (pattern.indexOf('.') !== -1) {
+                let patternParts = pattern.split('.')
+                patternExtension = '.' + patternParts.pop()
+                if (patternExtension.indexOf('*') !== -1) {
+                    patternExtension = ''
+                }
+            }
 
             filenames = filenames.filter(filename =>
                 filename.slice(-3) === '.js' ||
                 filename.slice(-3) === '.ts' ||
                 filename.slice(-8) === '.feature' ||
-                filename.slice(-7) === '.coffee')
+                filename.slice(-7) === '.coffee' ||
```

Comment:
Why not just add:

Reply:
@christian-bromann We can just use `path.extname(filename)`. What do you think?
`path` is already imported and also using `path.extname(filename)` should prevent us from mistakes when we pass the wrong number to `slice` function.

I'll create PR if you agree.

Reply:
@andriilazebnyi yes! Sounds good

Reply:
@christian-bromann Great! I'll prepare PR.

Reply:
@christian-bromann Please, see https://github.com/webdriverio/webdriverio/pull/2630